include/fmt/core.h

```diff
@@ -1613,6 +1629,194 @@ inline void print(const S& format_str, Args&&... args) {
       internal::make_args_checked<Args...>(format_str, args...));
 #endif
 }
+
+namespace internal {
+
+template <typename T, typename Char> struct is_string_view : std::false_type {};
+
+template <typename Char>
+struct is_string_view<basic_string_view<Char>, Char> : std::true_type {};
+
+#ifdef FMT_USE_STRING_VIEW
+template <typename Traits, typename Char>
+struct is_string_view<std::basic_string_view<Char, Traits>, Char>
+    : std::true_type {};
+#endif
+
+#ifdef FMT_USE_EXPERIMENTAL_STRING_VIEW
+template <typename Traits, typename Char>
+struct is_string_view<std::experimental::basic_string_view<Char, Traits>, Char>
+    : std::true_type {};
+#endif
+
+template <typename T> struct is_ref_wrapper : std::false_type {};
+
+template <typename T>
+struct is_ref_wrapper<std::reference_wrapper<T>> : std::true_type {};
```

Comment:
Why do we need to handle `std::reference_wrapper`?

Reply:
Hm, for me it's the main use-case :- ).
It's to let one to avoid dynamic memory allocation when the argument lifetime is guaranteed.

Actually I would want to make the `dynamic_format_arg_store` reusable and more friendly for those who concerned about memory allocations:
1. add reserve(); -- reserves 
2. add clean(); -- drops the `storage_` and resizes the `data_` to zero, but doesn't de-allocate the `storage_`.

Reply:
Makes sense. Thanks for clarification.

Reply:
Also let's rename this trait to `is_reference_wrapper` for consistency with the class name itself.